src/services/OswServiceTaskWiFi.cpp

```diff
@@ -173,21 +179,44 @@ bool OswServiceTaskWiFi::isWiFiEnabled() {
     return this->m_enableClient;
 }
 
+void OswServiceTaskWiFi::loadCredentials(uint8_t reconnectCount) {
+  switch (reconnectCount) {
+    case 1:
+      this->m_clientSSID = std::move(OswConfigAllKeys::wifiSsid.get());
```

Comment:
Do **NOT** use `std::move` here. It will **destroy** the data of the source (maybe you're lucky and Arduinos `String` class is too stupid for that, but...). DON'T.